scripts/build_renewable_profiles.py

```diff
@@ -301,6 +301,18 @@ def estimate_bus_loss(data_column, tech):
     return share_missed_buses
 
 
+def filter_cutout_region(cutout, regions):
```

Comment:
@fneum maybe interesting for PyPSA-Eur?
When you have a EU cutout, you will only create the renewable profiles in the country of interest